### PR TITLE
Fixed broken link

### PR DESCRIPTION
**Pull request by tkacholga**
Thank you for taking the time to work on a Pull Request for this project!

To ensure your PR is dealt with swiftly please check the following:

- [ ] Your submissions are formatted according to the guidelines in the [contributing guide](CONTRIBUTING.md)
- [ ] Your additions are ordered alphabetically
- [ ] Your submission has a useful description
- [ ] The description does not end with punctuation
- [ ] Each table column should be padded with one space on either side
- [ ] You have searched the repository for any relevant issues or pull requests
- [ ] Any category you are creating has the minimum requirement of 3 items
- [ ] All changes have been [squashed][squash-link] into a single commit

[squash-link]: <https://github.com/todotxt/todo.txt-android/wiki/Squash-All-Commits-Related-to-a-Single-Issue-into-a-Single-Commit>

### CODE REVIEW

**Comment by jbrooksuk** [comment]
Are these services no longer available?

**Comment by tkacholga** [comment]
Yes, that's right.

File: README.md [hunk 1]
```diff
@@ -212,6 +211,7 @@ API | Description | Auth | HTTPS | CORS |
 | [Poloniex](https://poloniex.com/support/api/) | US based digital asset exchange | `apiKey` | Yes | Unknown |
 | [WorldCoinIndex](https://www.worldcoinindex.com/apiservice) | Cryptocurrencies Prices | `apiKey` | Yes | Unknown |
 
+
```
Comment (yannbertrand) [inline README.md]:
```suggestion
```

**Comment by SuperKogito** [comment]
I think broken links are a common issue and a radical solution to this problem is automatically checking and reporting them. Luckily the new GitHub actions and  workflows help with that. I recently wrote an action that just does that. [URLs-checker](https://github.com/marketplace/actions/urls-checker) will check and report broken links, which is useful to spot similar issues to this one.

**Comment by matheusfelipeog** [comment]
The LectServe API is working and the Musikki and Pixhost APIs have already been removed

**Pull request closed without merging**